### ISSUE TITLE
Code of the models

### ISSUE THREAD

**Issue by fernandaassi** [MatthewMatero/AffectiveLanguageForecasting#2]
Hi! Do you have the code for the training/evaluation of the models described in the paper available somewhere? If you don't, can you send them to me? Thanks!

**Comment by MatthewMatero** [comment]
Hello,

The code available in the repository is enough to get up and running with a train/test cycle using the baseline (linear) autoregressive models. If you want to apply deep learning I would suggest using the supplied dataloader class and editing it to work with your favorite DL library. 

I have some old code that I used for the paper but this is written in PY2.7 and PyTorch 0.4 so it is really not that helpful since you will need to rewrite a bunch in newer Py3 and PyTorch 1.x versions. This old version of the code also does not work with the CSV files but rather a MySQL database I was working with when developing this work, which is why I also did not share it publicly. 

If you'd like I can share the model definitions (written in PyTorch 0.4.0) but I don't see the benefit of sharing the train/test loop code, since the new data_loader.py is a cleaner implementation of a lot of what I used when originally working on this, for the purpose of sharing publicly.
